jest/gen-config.js

```diff
@@ -56,7 +56,7 @@ var config = {
   moduleFileExtensions: ["js", "json", "es6"],
   modulePathIgnorePatterns: ["/tmp/", "/node_modules/", "/.module-cache/"],
   timers: "fake",
-  coverageReporters: ["json", "lcov", "cobertura", "text"],
+  coverageReporters: ["cobertura"],
```

Comment:
If you move the previous commit, you have to move this too.

Reply:
https://github.com/dcos/dcos-ui/pull/2801 is the PR for this now